redirect_report.rb

```diff
@@ -118,19 +147,39 @@ def run(args)
       end
     end
 
-    puts %w{public syndication google bing path}.join("\t")
-    $redirects.keys.sort { |a, b| $redirects[b] <=> $redirects[a] }
-      .each do |redirect|
-      puts [$redirects[redirect].public,
-            $redirects[redirect].syndication,
-            $redirects[redirect].google,
-            $redirects[redirect].bing,
-            redirect.source].join("\t")
+    redirects_sorted = $redirects.values.sort
+    if $config.mail_to_address
+      report_file = StringIO.new
+      format_output redirects_sorted, report_file, ','
+      mail_redirects_report $config.mail_to_address, report_file.string
+    else
+      format_output redirects_sorted, $stdout
+    end
+  end
+
+  def format_output(redirects, stream, delimit="\t")
+    stream.puts %w{public syndication google bing path}.join(delimit)
+    redirects.each do |redirect|
+      stream.puts [redirect.public,
+                   redirect.syndication,
+                   redirect.google,
+                   redirect.bing,
+                   redirect.regexp.source].join(delimit)
+    end
+  end
+
+  def mail_redirects_report(to_address, report_contents)
+    Mail.deliver do
+      from 'development.team@moneyadviceservice.org.uk'
```

Comment:
Well, we certainly could, but do you think this project warrants the effort? We won't be changing the From: in different environments, for example ...